providers/provider-pojo/src/main/java/org/apache/servicecomb/provider/pojo/Invoker.java

```diff
@@ -148,7 +148,7 @@ public Object invoke(Object proxy, Method method, Object[] args) {
     }
 
     PojoConsumerOperationMeta pojoConsumerOperationMeta = consumerMeta
-        .findOperationMeta(MethodUtils.findSwaggerMethodName(method));
+        .findOperationMeta(MethodUtils.findSwaggerMethodName(method), consumerIntf);
```

Comment:
consumerMeta only belongs to this invoker instance  
only relate to this one consumerIntf class  
seems no need to build a complex key?

Reply:
This is what this PR for , two interfaces should working in one consumer
```
interface DateTimeSchemaInf {
  Date getDate(Date date);	
}


interface DateTimeSchemaWithContextInf {
  Date getDate(InvocationContext context, Date date);
}
```

Reply:
meta from Invoker should not set in OperationMeta  
because as you said, there are different invokers  
we can create a new issue to optimize this.

Reply:
issue: https://issues.apache.org/jira/browse/SCB-1830